app/mailers/message_mailer.rb

```diff
@@ -3,7 +3,9 @@ class MessageMailer < ApplicationMailer
 	default :to => "dave@bigtalker.io"
```

Comment:
@slnwlf Dave, do you use this email or the Gmail, use only once, so it will be easy to check. @laurakathleen if Dave use the Gmail, just change this tho ENV['BCC_EMAIL']